### PR TITLE
TST-123 This is a note that dw-test is a test

### PR DESCRIPTION
**Pull request by dulanism**

### CODE REVIEW

**Comment by joedragons** [comment]
Received.

File: chapter-introduction.asciidoc [hunk 1]
```diff
@@ -2,7 +2,7 @@
 == Introducing {pro}
 
 [[intro-sect-intro]]
-=== Introduction
+=== Introduction - this is a test-dw
```
Comment (jwayman) [inline chapter-introduction.asciidoc]: Needs more commas per @kellyrob99 .

**Comment by mosabua** [comment]
Descriptions should always have a linked to the JIRA issue and the bamboo branch build ;-)

**Comment by mosabua** [comment]
And some sort of description.

**Comment by mosabua** [comment]
PR should also use labels... Like I applied.

File: chapter-introduction.asciidoc (hunk 1 above)
Comment (kellyrob99) [inline chapter-introduction.asciidoc]: The correct number of commas is none even if it makes sentences seem kinda long and run-on like maybe possibly this one :)

**Comment by dulanism** [comment]
Noted @mosabua. Thx

**Comment by mosabua** [comment]
Also note that you would merge to master with this PR ... which is Nexus 2.12 at the moment. You probably will want to create a branch off nexus-3.0.x and the PR into that branch..

File: chapter-introduction.asciidoc (hunk 1 above)
Comment (dulanism) [inline chapter-introduction.asciidoc]: @kellyrob99 @jwayman PWNED --- > me

**Pull request closed without merging**